### PR TITLE
15021 filter by date range

### PR DESCRIPTION
**Pull request by sgrg1**
date range filter has been added to react-filter-list

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
-        content: "-";
-      }
+          .dg_date-btn1 {
+            display: block;
+            position: relative;
+            right: -230px;
+            top: -35px;
+            cursor: pointer;
+          }
     </style>
```
Comment (tmccoy529) [inline public/index.html]: The styles from master need to come back. Those are used internal to this project. 

 <style>	    <style>
      .plus-text-icon,	         .dg_date-container1  {
      .minus-text-icon {	
        position: relative;	        position: relative;
      }	        min-width: 40%;
      .plus-text-icon:before,	        margin-right:20px;
      .minus-text-icon:before {	         }
        display: inline-block;	
        margin-right: 5px;	
      }	


      .plus-text-icon:before {	          .dg_date-btn1 {
        content: "+";	            display: block;
      }	            position: relative;

            right: -230px;
      .minus-text-icon:before {	            top: -35px;
        content: "-";	            cursor: pointer;
      }	          }
    </style>

Just cope that back over the style tag in your branch

File: rollup.config.js [hunk 2]
```diff
@@ -17,14 +23,23 @@ const plugins = [
...
+
 export default [
+  // { 
+  //   input:'./src/components/entry.js',
+  // output: {file:'bundle.js', format:'cjs'},
+  // plugins: [
+  //   css({ output: 'bundle.css' })
+  // ]},
   {
     input: "demo/index.js",
```
Comment (tmccoy529) [inline rollup.config.js:24]: you can delete the commented code. We wont need it now that its been moved to the component library.

File: src/components/FilterList.jsx [hunk 3]
```diff
@@ -13,6 +16,9 @@ import Filters from "./Filters.jsx";
 import PropTypes from "prop-types";
 import RecordsMessage from "./RecordsMessage";
 import { withRouter } from "react-router-dom";
+import FilterDateSelector from "./FilterDateRange";
+import { Button } from "@baltimorecounty/dotgov-components";
+import { get } from "http";
```
Comment (tmccoy529) [inline src/components/FilterList.jsx:21]: that get import isnt being used and can be deleted.

File: rollup.config.js (hunk 2 above)
Comment (sgrg1) [reply rollup.config.js:24]: done

File: src/components/FilterList.jsx (hunk 3 above)
Comment (sgrg1) [reply src/components/FilterList.jsx:21]: done

**Review by tmccoy529 — APPROVED** [review]

**Pull request merged**